### ISSUE TITLE
Not working on Ubuntu 18.04 since v0.0.2

### ISSUE THREAD

**Issue by ColonelJ** [fbsobreira/TronLedger#2]
Hi, the Linux (.deb) versions have not been working since v0.0.2, since then all you get is a blank (black) window when you start the app.

**Comment by MichaelMichaelMichaelMichaelMichael** [comment]
same here on ubuntu 17.10

**Comment by fbsobreira** [comment]
Hi, we are merging TronLedger integration with TronWallet Play. TronLedger will be deprecated, as we can offer a better support with TronWallet play.
Please check last linux version here: https://tronwallet-play-releases.s3.amazonaws.com/TronWalletPlay+1.1.0.AppImage

**Comment by ColonelJ** [comment]
I tried that version when it came out, and still it has the same issue with a blank window.  I reported it as a bug following their link and filling out a form, but I have no way to track progress on the bug, and haven't received any updates or communication from them about it.  Their support page is nothing but a how to/FAQ, so I don't see how their support is 'better'.  Any improvement on this situation would be appreciated.

**Comment by fbsobreira** [comment]
I will check possibility of open a git repository for issue track on
TWplay. The blank screen on TWplay is when opening? Ubuntu 18? I will ask
the team for access to this report. Thanks

**Comment by ColonelJ** [comment]
Yeah, when you open the app (in Ubuntu 18.04) you get a window which is blank except for 'TronWalletPlay' written in the top left and minimize/maximize/close buttons in the top right (they work).  (The first time I opened the app I got a dialog box before this screen asking if I wanted to install, but never seen that again since.)  Thanks for your help.

**Comment by fbsobreira** [comment]
New release on TWPlay. Could you please try if the error persists?
https://tronwallet-play-releases.s3.amazonaws.com/TronWalletPlay+1.1.2.AppImage

**Comment by ColonelJ** [comment]
FYI, the auto updater in the old version tried to download version 1.1.1 and got a 404 (this is shown in the terminal when running it from there - the window was still blank as always).

Ignoring that, I tried this new version and it seems to work!  I managed to freeze my rewards, submit new votes, send some BTT to another account, and then I even was able to go on the TronWatchMarket and buy 10TWX with the fractions of TRX I had left.  Very impressive!  However, I did get a lot of errors along the way (e.g. when 'Allow Data' wasn't enabled, and then two further errors when I tried that again, including one that said it had too many signatures - had to close the dialog and start again with freezing for it to work) and the UX was not perfect (e.g. no easy way to see your current votes).  Obviously I have the 'Allow data' and 'Custom contracts' options enabled on the Ledger's TRON app now.

Thanks again for whatever you did to get it to work, and I hope it will continue working in future versions.  If I have a better place to post issues I have with the app then I would happily be able to give more detailed feedback.
